google/ggrc-core
# GGRC-8814 Disable ability for opening create modal for Org Groups

# Issue description

Creating of Org Group disabled after migration, "+ Create" button should be deleted from autocomplete-results component for Audit firm field.

# Steps to test the changes

1. Log in GGRC
2. Start creating a new Audit
3. Type non existing org group in Audit Firm field
4. Review
5. Expected: Only "No results" displays.

# Solution description

Delete +create button from autocomplete-results template for Org Groups.

# Sanity checklist




# PR Review checklist
